vhdbuilder/cgmanifest.json

```diff
@@ -5,8 +5,8 @@
                 "Type": "other",
                 "Other": {
                     "Name": "CalicoWindows",
-                    "Version": "3.17.1",
-                    "DownloadUrl": "https://acs-mirror.azureedge.net/calico-node/v3.17.1/binaries/calico-windows-v3.17.1.zip"
+                    "Version": "3.19.0",
+                    "DownloadUrl": "https://acs-mirror.azureedge.net/calico-node/v3.19.0/binaries/calico-windows-v3.19.0.zip"
```

Comment:
@mainred, I think that we do not need to change it every time since the license should never be changed. I change it since you change containerd in last pr.

Reply:
Hi @AbelHu 
This is from CELA team. We may not able to register all oss components but we can try to make sure what has been done are well registered with different versions we are using in the vhd. I think it's not too much effort.
_Correct – all OSS is registered and that includes use of different versions of a component._